terraform/api_gateway.tf.json.template.py

```diff
@@ -9,6 +10,16 @@
     emit_tf,
 )
 from azul.objects import InternMeta
+from azul.indexer_policy import indexer_policy
```

Comment:
Can this be

```suggestion
import azul.indexer.lambda_policy as indexer_policy
```

I know the `azul.indexer` package doesn't currently exist but indexer-specific code should be moved into it in the future. This could be the start of that.

Same for `service` except that that package already exists.

Reply:
Yes, I'll make that change.